lib/app/endpoint/home.rb

```diff
@@ -164,46 +171,45 @@ class Home < Endpoint
           end
         end
 
-        # Handles EHR Launch and any redirects
         get '/:id/:key/:endpoint/?' do
           instance = Inferno::Models::TestingInstance.get(params[:id])
-          halt 404 unless !instance.nil? && instance.client_endpoint_key == params[:key] && %w[launch redirect].include?(params[:endpoint])
+          halt 404 unless !instance.nil? && instance.client_endpoint_key == params[:key] && ['launch','redirect'].include?(params[:endpoint])
 
           sequence_result = instance.waiting_on_sequence
 
           if sequence_result.nil? || sequence_result.result != 'wait'
-            redirect "/#{base_path}/#{params[:id]}/?error=no_#{params[:endpoint]}"
+            redirect "/#{BASE_PATH}/#{params[:id]}/?error=no_#{params[:endpoint]}"
```

Comment:
Recommend leaving this with the `base_path` helper method provided in `configuration.rb`:
https://github.com/siteadmin/inferno/blob/ccef0fbef3856ad1f1a1787f3e9bb6288cdbad54/lib/app/helpers/configuration.rb#L8-L10
```suggestion
            redirect "/#{base_path}/#{params[:id]}/?error=no_#{params[:endpoint]}"
```